src/stores/AppSettingsStore.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import type RootStore from './RootStore';
+
+import { AsyncStorage } from 'react-native';
+import { action, computed, observable, runInAction } from 'mobx';
+
+const APP_SETTINGS_STORAGE_KEY = 'app_settings';
+
+type AppSettings = {|
+  manageTapsEnabled: boolean,
+  multiAccountModeEnabled: boolean,
+|};
+
+// todo somehow save settings for particular user and restore them after relogin?
+// or save user settings on the server?
```

Comment:
what do you think about this? 
When user login out -> we should reset all stores state and when he login we want to have restored app settings for that user, is this makes sense to keep app settings for users on the server?

Reply:
This scenario will rarely happen so I don't want to store it on the server. What I did in the ionic app is have a Map keyed with the userID with a value of the settings which was stored in localstorage.